tests/testthat/test-correlation.R

```diff
@@ -237,6 +242,16 @@ test_that("correlation returns an appropriately structured result for assay agai
   #expect_equal(as.character(unique(statsData$data1)), names(testOTU)[2:(length(names(testOTU)) - 1)])
   #expect_equal(as.character(unique(statsData$data2)), names(testOTU)[3:length(names(testOTU))])
   expect_true(all(!is.na(statsData)))
+
+  # method = sparcc
+  result <- selfCorrelation(data, method='sparcc', verbose = FALSE)
+  statsData <- result@statistics@statistics
+  expect_s3_class(statsData, 'data.frame')
+  expect_equal(names(statsData), c('data1','data2','correlationCoef','pValue'))
+  expect_equal(nrow(statsData), 66) # Should be number of taxa * number of taxa
+  #expect_equal(as.character(unique(statsData$data1)), names(testOTU)[2:(length(names(testOTU)) - 1)])
```

Comment:
Can we delete these or reimplement the tests? Not sure why they would be failing

Reply:
if we prefilter, we no longer know which features will make it into the result. i suppose i could explicitly set all the prefilters to 0 here though